lib/database/index.ts

```diff
@@ -258,6 +264,13 @@ export interface PgStacDatabaseProps extends rds.DatabaseInstanceProps {
    */
   readonly addPgbouncer?: boolean;
 
+  /**
+   * Properties for the pgbouncer ec2 instance
+   *
+   * @default - defined in the construct
+   */
+  readonly pgbouncerInstanceProps?: ec2.InstanceProps | any;
```

Comment:
I had to do this to allow users to pass any incomplete set of parameters for the ec2.InstanceProps because jsii will not let you use Partial<ec2.InstanceProps> which would have been nice.